packages/test/test-end-to-end-tests/src/test/entryPointCompat.spec.ts

```diff
@@ -65,9 +65,12 @@ describe("entryPoint compat", () => {
 	});
 
 	const loaderWithRequest = "2.0.0-internal.7.0.0";
-	describeInstallVersions({
-		requestAbsoluteVersions: [loaderWithRequest],
-	})("loader compat", (_) => {
+	describeInstallVersions(
+		{
+			requestAbsoluteVersions: [loaderWithRequest],
+		},
+		180000, // timeoutMs 3 minutes
```

Comment:
I think it would be better to increase the default timeout that `describeInstallVersions` uses for its `before` hook that installs the modules. This should limit the scope of longer tests. As-is, I believe if all the tests in this suite hung, we could hit CI limits on job time.

Reply:
looks like that's in `describeWithVersions.ts` ~L68